dev.Dockerfile

```diff
@@ -51,7 +51,7 @@ RUN sudo apt-get install -y vim lld
 RUN mkdir -p ~/.cargo
 # Make cargo put compile artifacts in non-bind-mounted directory
 # To re-use compiled artifacts, mount a docker volume to /tmp/target
-RUN echo -e '[build]\ntarget-dir="/tmp/target"' > ~/.cargo/config.toml
+RUN echo -e '[build]\ntarget-dir="/tmp/target"\njobs=2' > ~/.cargo/config.toml
```

Comment:
can we add a comment above for why jobs =2

Reply:
done.